File: Data Structures & Algorithms/two-integer-sum/submission-0.cpp
#include <unordered_set>
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
    int diff = {0};
    unordered_set<int> seen;
    for (int i = 0; i < nums.size(); ++i ){
        diff = target - nums[i];
        if (seen.count(diff)) return {i, seen[diff]};
        seen[nums[i]] = i;
    }
    return {};
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        map = {}

        for i in nums:
            map[i] = 1 + map.get(nums[i], 0)

        for i in map:
            if map[i] >= k:
                return nums[i]

        return []
        